pages/landing/components/LandingPageCard.js

```diff
@@ -1,8 +1,42 @@
 import LandingCardButton from "./LandingCardButton";
 import LandingCardContent from "./LandingCardContent";
 import HorizontalLine from "./HorizontalLine";
+import Cookies from "js-cookie";
+import { useSnackbar } from "notistack";
+import { useRouter } from "next/router";
+import axios from "axios";
+import React, { useContext, useEffect, useState } from "react";
 
 function LandingPageCard(props) {
+  const { enqueueSnackbar, closeSnackbar } = useSnackbar();
+
+  const acceptSite = async () => {
```

Comment:
There is no use of this function on this page so just remove it

Reply:
Yes sir deleted it